processing/src/main/java/io/druid/jackson/DefaultObjectMapper.java

```diff
@@ -53,6 +53,7 @@ public DefaultObjectMapper(JsonFactory factory)
     configure(MapperFeature.AUTO_DETECT_FIELDS, false);
     configure(MapperFeature.AUTO_DETECT_IS_GETTERS, false);
     configure(MapperFeature.AUTO_DETECT_SETTERS, false);
+    configure(MapperFeature.ALLOW_FINAL_FIELDS_AS_MUTATORS, false);
     configure(SerializationFeature.INDENT_OUTPUT, false);
```

Comment:
My reason for wanting this field is because if a field is assumed final and the constructor uses it, then it is potentially later set to something else, that is an issue.